modules/tinymce/src/plugins/help/main/ts/ui/KeyboardNavTab.ts

```diff
@@ -3,68 +3,91 @@ import { Dialog } from 'tinymce/core/api/ui/Ui';
 // TODO: When we translate this, we should pull the HTML out into a HTML file, the way TBIO does it.
 // That requires webpack and rollup changes though, so inlining it for now.
 /* eslint-disable max-len */
-const description = `<h1>Editor UI keyboard navigation</h1>
+const description = `<h1>Activate keyboard navigation</h1>
```

Comment:
Activate is a technical term that implies some element is made to be in an `active` state. This is not what we mean here. "Activating keyboard navigation" from the previous text tells the user how to start/begin navigating the UI with the keyboard, so I think it is best to change it to something like that.
```suggestion
const description = `<h1>Begin keyboard navigation</h1>
```